wcivf/apps/people/models.py

```diff
@@ -189,12 +189,29 @@ def linkedin_username(self):
     @property
     def youtube_username(self):
         youtube_url = self.youtube_profile
-        if ("channel") in youtube_url:
+        if "channel" in youtube_url:
             return self.name + "'s Channel"
-        else:
-            youtube_split = list(filter(None, youtube_url.split("/")))
-            youtube_username = youtube_split[-1]
-            return youtube_username
+        youtube_split = list(filter(None, youtube_url.split("/")))
+        youtube_username = youtube_split[-1]
+        return youtube_username
+
+    @property
+    def long_statement(self):
+        return self.statement_count > 10
```

Comment:
10 words is a fairly short statement I think - maybe this should be more like 100? Needs to be tested with some real content I expect.